runtime/frequency/src/lib.rs

```diff
@@ -249,7 +249,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 54,
+	spec_version: 55,
```

Comment:
Why the spec bump?

Reply:
The metadata comparison in the GH action is showing a difference and is requiring it.

Reply:
This is because extrinsic documentation is part of the metadata.